src/core/core.controller.js

```diff
@@ -148,13 +148,34 @@ function onAnimationProgress(ctx) {
 	helpers.callback(animationOptions && animationOptions.onProgress, arguments, chart);
 }
 
+/**
+ * Chart.js can take a string id of a canvas element, a 2d context, or a canvas element itself.
+ * Attempt to unwrap the item passed into the chart constructor so that it is a canvas element (if possible).
+ */
+function unwrapItem(item) {
```

Comment:
"item" is a bit generic. Maybe `unwrapCanvas`. Or `getCanvas` since it's maybe not being "unwrapped" in the case that it's a dom id